docs/Installation.md

```diff
@@ -671,20 +637,21 @@ Install, configure and start database engine:
   sudo systemctl start postgresql
   ```
 
-Initialize Zonemaster database and user (unless you keep an old database):
+To create the database and the database user (unless you keep an old database)
+run the command on one line.
 
 ```sh
-sudo -u postgres createuser -P zonemaster
+sudo -u postgres psql -c "CREATE USER zonemaster WITH PASSWORD 'zonemaster';"  -c "CREATE DATABASE zonemaster WITH ENCODING 'UTF8';"

```

Comment:
We need to define the owner of the database, and would it be possible to split the command in 2 commands?
```
sudo -u postgres psql -c "CREATE USER zonemaster WITH PASSWORD 'zonemaster';"
sudo -u postgres psql -c "CREATE DATABASE zonemaster WITH OWNER 'zonemaster' ENCODING 'UTF8';"
```

Reply:
Same question about sudo as above. 

I have added "WITH OWNER 'zonemaster'" but it was not in the FreeBSD commend either, and it works anyway. Have you tested without? I will test with.

Reply:
I have added "WITH OWNER 'zonemaster'" for all OSs on your suggestion, but I also noted that I ran the following command under FreeBSD when testing (*note* no "owner zonemaster"):
```
psql -U postgres -c "CREATE USER zonemaster WITH PASSWORD 'zonemaster';" -c "CREATE DATABASE zonemaster WITH ENCODING 'UTF8';"
```
There was no error when creating, and there was no error when I ran several tests with that database, so my implicit question was is that setting really needed?

It is not important though.